aweber.com.landing-pages-subdomain.json

```diff
@@ -0,0 +1,18 @@
+{
+   "providerId":"aweber.com",
+   "providerName":"AWeber",
+   "serviceId":"landing-pages-subdomain",
+   "serviceName":"AWeber Landing Pages",
+   "version": 1,
+   "logoUrl":"https://assets.aweber-static.com/page-templates/assets/img/powered_by.png",
+   "description":"Configure a domain to AWeber Landing Pages",
+   "variableDescription":"",
+   "records":[
+       {
+         "type": "A",
+         "host": "{%host%}",
```

Comment:
See previous comment. `@` is the proper value.

Reply:
ok thank you for the feedback and guidance.